### PR TITLE
Fixed release source

### PR DESCRIPTION
**Pull request by SteveHobs**
Changed from bitbucket to github

### CODE REVIEW

File: automatic/keepass-plugin-readablepassphrasegen/update.ps1 [hunk 1]
```diff
@@ -25,16 +25,15 @@ function global:au_AfterUpdate {
 
 function global:au_GetLatest {
     $page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $regexUrl = 'Version (?<version>[\d\.]+)'
+    $regexUrl = 'ReadablePassphrase\.(.*)\.plgx$'
```
Comment (pauby) [inline automatic/keepass-plugin-readablepassphrasegen/update.ps1:28]: I like to use named index when looking at matches than numbered ones. Please see the previous code to use for that. Can you also fix the regular expression to match on . and digits only as the previous one did.

File: automatic/keepass-plugin-readablepassphrasegen/update.ps1 [hunk 2]
```diff
@@ -25,16 +25,15 @@ function global:au_AfterUpdate {
...
 
-    $null = $page.content -match $regexUrl
-    $version = $matches.version
-    $url = "https://bitbucket.org/ligos/readablepassphrasegenerator/downloads/ReadablePassphrase%20$version.plgx"
+    $url = $page.links | Where-Object href -match $regexUrl | Select-Object -first 1 -expand href
+    $version = $matches[1]
```
Comment (pauby) [inline automatic/keepass-plugin-readablepassphrasegen/update.ps1:31]: Same as previous comment.

Comment (SteveHobs) [reply automatic/keepass-plugin-readablepassphrasegen/update.ps1]: Cheers, updated the regex

**Comment by pauby** [comment]
Awesome work! Thanks for your contribution!

**Pull request merged**